cairocffi/test_xcb.py

```diff
@@ -149,6 +155,11 @@ def test_xcb_pixmap(xcb_conn):
         width, height
     )
 
+    ctx = None
+    surface = None
+    remove_gc(xcb_conn, gc)
+    remove_pixmap(xcb_conn, pixmap)
```

Comment:
Is this something that users always need to do? If so, should it happen in the library rather than in tests?